docker-compose.yml

```diff
@@ -0,0 +1,100 @@
+services:
+
+  base:
+    image: ${REGISTRY}/inginious/core-base:${VERSION}
```

Comment:
Maybe it would be interesting to set the default `REGISTRY` and `VERSION` values here, or in a `.env` file.

=> `${REGISTRY:-ghcr.io}` and `${VERSION:-latest}`

so that it doesn't need to be passed as environment variables in the default case.

Reply:
I was thinking the same thing yesterday while writing some documentation. Will update that.